### PR TITLE
'reload permissions' command & ReloadedPlugins event

### PR DESCRIPTION
**Pull request by Ficus-x**

### CODE REVIEW

File: Exiled.Events/Commands/Reload/Permissions.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out string response)
+        {
+            if (!sender.CheckPermission("ee.reloadpermissions"))
+            {
+                response = "You can't reload permissions, you don't have \"ee.reloadpermissions\" permission.";
+                return false;
+            }
+
+            Exiled.Permissions.Extensions.Permissions.Reload();
+            Server.OnReloadedPermissions();
```
Comment (iopietro) [inline Exiled.Events/Commands/Reload/Permissions.cs:45]: You should move this call at the end of the `Permission::Reload` method.

File: Exiled.Events/Commands/Reload/Plugins.cs [hunk 2]
```diff
@@ -45,6 +46,8 @@ public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out s
 
             Loader.ReloadPlugins();
```
Comment (iopietro) [inline Exiled.Events/Commands/Reload/Plugins.cs:47]: Same as above.

**Review by iopietro — CHANGES_REQUESTED** [review]

File: Exiled.Events/Commands/Reload/Permissions.cs (hunk 1 above)
Comment (Ficus-x) [reply Exiled.Events/Commands/Reload/Permissions.cs:45]: Check the summaries of OnReloaded... events

Comment (Ficus-x) [reply Exiled.Events/Commands/Reload/Permissions.cs:45]: Other events of reloaded something calls in command too

Comment (iopietro) [reply Exiled.Events/Commands/Reload/Permissions.cs:45]: I thought they were generic events, I don't know what could be the use case of those events, they're fine but I would have liked them more if made generic.

**Pull request merged**